src/root.component.tsx

```diff
@@ -8,6 +8,76 @@ defineConfigSchema("@openmrs/esm-home", {
   buttons: {
     enabled: {
       default: false
```

Comment:
I think set this to true for now

Reply:
Per my [versioning recommendation](https://github.com/openmrs/openmrs-rfc-frontend/pull/22/files), I'd like to release a minor version with it disabled, and then turn it on in the next major version. Does that seem reasonable?